### DISCUSSION TITLE
Despite what is written about the installation for Linux

### DISCUSSION THREAD

**Discussion by RedSunrise** [mitmproxy/mitmproxy#7285] [General]
I would like to see a list of the latest "stable" versions of mitmproxy for the most popular Linux distributions, which can be installed through package tools. I can write this list with instructions by myself if needed, but i need your opinion on it.

**Comment by mhils** [comment]
Encouraging the use of older mitmproxy versions creates a significant
support burden for us. So unless we have someone who's very actively
involved in maintenance and who would like to take on that additional load,
I'm opposed to any step in that direction. I'd be happy to add more docs on
e.g. using our Docker images, which are always current.
